README.md

```diff
@@ -2,15 +2,130 @@
 
 # AutoStep
 
-AutoStep is a new way of writing Gherkin-style tests, that reduces the requirement on programmers to be heavily involved in writing tests.
+AutoStep is a compiler and runner for Gherkin-style BDD tests, with a few extra language features
```

Comment:
Suggest you say "some extra" rather than "a few" as that sounds quite limited.

Reply:
Done, updated.